docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-ruby.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Build a Ruby application that uses YSQL
+headerTitle: Build a Ruby application
+description: Build a small Ruby application using the Ruby Pg driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: Ruby
+    identifier: cloud-ruby
+    weight: 700
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [Ruby application](https://github.com/yugabyte/yugabyte-simple-ruby-app) that connects to a YugabyteDB cluster using the [Ruby Pg driver](../../../../reference/drivers/ysql-client-drivers/#pg) and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in Ruby.
+
+## Prerequisites
+
+- Ruby 3.1 or later.
+- OpenSSL 1.1.1 or later (used by libpq and pg to establish secure SSL connections).
+- [libpq](../../../../reference/drivers/ysql-client-drivers/#libpq). Homebrew users on macOS can install using `brew install libpq`. You can download the PostgreSQL binaries and source from [PostgreSQL Downloads](https://www.postgresql.org/download/).
+- [Ruby pg](../../../../reference/drivers/ysql-client-drivers/#pg).
```

Comment:
Include the gem install command to install ruby pg here rather than under the Build and run the application process, because clicking the ruby pg link here directs me to this [page](https://deploy-preview-11479--infallible-bardeen-164bc9.netlify.app/latest/reference/drivers/ysql-client-drivers/#install-the-pg-driver) where the install command is for local installs.